applications/climatology/climatology.ipynb

```diff
@@ -0,0 +1,3533 @@
+{
```

Comment:
<p>And then before this cell, add a subhead for "Visualization".</p><p> </p><p>Also, this is picky, but I think the comment here could be a little more clear. I'd suggest something like:</p>

<pre class="ql-syntax" spellcheck="false"># Plot the first time slice of the calculated anomalies
tos_anom.isel(time=0).plot()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>